httpevents/request_safe.go

```diff
@@ -0,0 +1,15 @@
+// +build !go1.5
+
+package httpevents
+
+func bytesToStringNonEmpty(b []byte) string {
```

Comment:
Does this need to be it's own function? bytesToString is functionally the same I think (with an extra if check).